include/taskolib/execute_lua_script.h

```diff
@@ -26,7 +26,6 @@
 #define TASKOLIB_EXECUTE_LUA_SCRIPT_H_
 
 #include <string>
-#include <variant>
 
 #include <gul14/expected.h>
```

Comment:
We already used `expected` here? :grimacing:

Reply:
Yes, for the other function that we just added. :)